packages/typescript/src/mutator/BooleanLiteralMutator.ts

```diff
@@ -0,0 +1,25 @@
+import * as ts from 'typescript';
+
+import { printNode } from '../helpers/tsHelpers';
+
+import NodeMutator, { NodeReplacement } from './NodeMutator';
+
+export default class BooleanLiteralMutator extends NodeMutator<ts.BooleanLiteral> {
+  public name: string = 'BooleanLiteral';
+
+  public guard(node: ts.Node): node is ts.BooleanLiteral {
```

Comment:
Add `| ts.PrefixUnaryExpression` as return type. Currious why this wasn't a compile error :/